CONTRIBUTING.md

```diff
@@ -21,71 +21,69 @@ See [GitHub's official documentation](https://help.github.com/articles/using-pul
 
 ## Getting Started
 
-This repo contains source code, package build specifications for building the source into VI packages, and examples showing how to write and run MeasurementLink services.
+This repo contains source code, package build specifications for building the source into VI packages, and examples showing how to write and run measurement plug-ins.
 
 The source code can be found under the `Source` directory. The package definition files (.vipb) can be found under the `Build Specs` directory. The example code can be found under the `Example Measurements` directory.
 
 ## Building the LabVIEW Packages
 
 The source code is built into five LabVIEW packages
 
-* `ni_measurementlink_service`
-* `ni_measurementlink_generator`
+* `ni_measurement_plugin_sdk`
+* `ni_measurement_plug_in_generator`
```

Comment:
I've not been using a space in `plugin` for package names.

Reply:
Updated along with other package name updates. But note that, for example, the <Package_File_Name> in Measurement Plug-In Generator.vipb currently has an underscore in there: NI_Measurement_Plug_In_Generator

Reply:
I'm not seeing that in main. Is there somewhere else I'm missing?
https://github.com/ni/measurement-plugin-labview/blob/main/Source/Build%20Specs/Measurement%20Plug-In%20SDK%20Generator.vipb#L3

Reply:
Ah, I see now. I'm [one commit behind](https://github.com/DavidCurtiss/measurement-plugin-labview/compare/users/dcurtiss/md-renames...ni%3Ameasurement-plugin-labview%3Amain) main.